incubator/java-openliberty/README.md

```diff
@@ -115,6 +116,68 @@ The command `appsody test` launches the Open Liberty server, runs integration te
 * At the time of the release of this java-openliberty stack, this problem seems to be getting the active attention of the Docker Desktop for Windows developement team, (e.g. see [this issue](https://github.com/docker/for-win/issues/5530)). Naturally, updating your Docker Desktop for Windows installation might help, however, we can not simply point to a recommended version that is known to work for all users.   
 * **Workaround**: This may be worked around by making the changes from the host, and then doing a `touch` of the corresponding files from within the container.
 
+## Application Deployment 
+
+### Enabling secure endpoints in deployment
+
+Default security credentials are provided for local development. These default values are removed from the official deployment image which simply exposes username and password credentials as environment variables.
+To enable secure endpoints in deployment (e.g. /metrics) perform the following steps:
+
+1. Enable the HTTPS port
+
+   Under `spec` in the app-deploy.yaml for your application:
+   
+   1. Change `port: 9080` to `port: 9443` 
+   1. Add the following:
+
+      ```bash
+      route:
+        termination: passthrough
+      ```
+   
+1. Set security credentials via a pre-existing Secret
+
+   The username and password credentials for the application are exposed via environment variables. You can set these variables using a Secret by adding the following where `mySecret` is the name of the pre-existing Secret in your namespace:
+
+   ```bash
+   env:
+     - name: STACK_USERNAME
+     valueFrom:
+       secretKeyRef:
+         key: username
+         name: mySecret
+     - name: STACK_PASSWORD
+     valueFrom:
+       secretKeyRef:
+         key: password
+         name: mySecret
+   ```
+
+   This will set environment variables in the deployment called `STACK_USERNAME` and `STACK_PASSWORD` which map to the username and password of the "mySecret" Secret respectively. 
```

Comment:
of the "mySecret" Secret 
  =>
of `mySecret`, respectively.

(You mentioned it's a Secret enough I think).